README.md

```diff
@@ -939,6 +940,12 @@ registered with cpptrace.
 
 [jitci]: https://sourceware.org/gdb/current/onlinedocs/gdb.html/JIT-Interface.html
 
+## Loading Libraries at Runtime
+
+When loading libraries at runtime, e.g. with `dlopen` or `LoadLibrary`, 
```

Comment:
Since this isn't needed for linux/macos I believe, it'd be good to specify that this is only needed on windows for `LoadLibrary`